WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2328,6 +2340,45 @@ public void onMediaUploadCancelClicked(String localMediaId) {
     }
 
     @Override
+    public void onMediaDeleted(String localMediaId) {
+        if (!TextUtils.isEmpty(localMediaId)) {
```

Comment:
Good observation @nbradbury  - that's because in several places we obtain the id from `wp-id` class attribute within the Post content, which is text.